src/main/java/com/shootdoori/match/entity/TeamType.java

```diff
@@ -0,0 +1,15 @@
+package com.shootdoori.match.entity;
+
+public enum TeamType {
+    중앙동아리,
+    과동아리,
+    기타;
+
+    public static TeamType from(String value) {
+        return switch (value) {
+            case "중앙동아리" -> TeamType.중앙동아리;
+            case "과동아리" -> TeamType.과동아리;
+            default -> TeamType.기타; // null 또는 매칭 실패 시 기본값
+        };
+    }
```

Comment:
```suggestion
public enum TeamType {
    CENTRAL_CLUB("중앙동아리"),
    DEPARTMENT_CLUB("과동아리"), 
    OTHER("기타");
    
    private final String displayName;
    
    TeamType(String displayName) {
        this.displayName = displayName;
    }
    
    public String getDisplayName() {
        return displayName;
    }
    
    public static TeamType fromDisplayName(String displayName) {
        for (TeamType type : values()) {
            if (type.displayName.equals(displayName)) {
                return type;
            }
        }
        throw new IllegalArgumentException("Unknown team type: " + displayName);
    }
```
- 구글 자바 스타일 가이드에서는 enum 상수는 영문 대문자와 언더스코어를 사용할 것은 권장하고 있어요.
- 기타라는 모호한 카테고리는 비즈니스 로직에서 예외 처리를 어렵게 만듭니다.